core/Cargo.toml

```diff
@@ -27,6 +27,8 @@ dex = []
 dev-telemetry = ["telemetry", "iroha_telemetry/dev-telemetry"]
 # Support Prometheus metrics. See https://prometheus.io/.
 expensive-telemetry = ["iroha_telemetry/metric-instrumentation"]
+# Support for mocking World object
+mock_world = ["mockall"]
```

Comment:
can we name it just `mock`? so that later we can add other mocks, not only `World`

Reply:
I think that if you're mocking an object defined in a create you depend on, the proper way is to make a trait inside the current crate and make you're mock implementation there, not in the dependency crate. Exporting cargo feature that defines mocked objects seems like a misuse of the cargo. In this case it would mean that all mock would be defined in the `iroha_client`. This doesn't mean that you cannot create mock objects inside `iroha_core`, it's just that those will only be usable in unit tests of that crate